### ISSUE TITLE
Support new OSs - CentOS Stream 10 - Update allocator module

### ISSUE THREAD

**Issue by jotacarma90** [wazuh/wazuh#24493]
# Description
| Related issue | Epic issue |
|---|---|
| https://github.com/wazuh/wazuh/issues/24128 | https://github.com/wazuh/wazuh/issues/24490 |

| Agent tier | Central components support | OS type |
|-|-|-|
| 1 | No | Major |

# Plan

**DevOps**
- [ ] **No central components**: Deploy an All In One (in our featured OS, probably Amazon Linux) and allocate an accessible machine with the new OS to test the agent. This is the first step for everything else.
- [ ] Add support in the allocator.
- [ ] Adapt Puppet.
- [ ] Adapt Ansible.
- [ ] Update AMI, OVA, or Docker images if needed.

**Comment by Enaraque** [comment]
# Update report

This version has not been officially released yet. When attempting to install the[ ISO](https://mirror.stream.centos.org/10-stream/BaseOS/x86_64/iso/) provided by CentOS on a VM, a "kernel panic" error occurs, making it impossible to test this version at this time.

This issue is marked as blocked until the official version has been released.

**Comment by teddytpc1** [comment]
This issue will be closed as not planned because there are no Vagrant boxes nor AMIs to deploy an instance.

**Comment by mjcr99** [comment]
[Official Release](https://www.centos.org/download/) available, reopening the issue.

**Comment by teddytpc1** [comment]
The CentOS Steam 10 Vagrant box is available: https://portal.cloud.hashicorp.com/vagrant/discover/centos/stream10
There's no official AMI in AWS yet.

**Comment by Enaraque** [comment]
# Update report

When using the official box, it has not been possible to bring up the VM successfully.

![Image](https://github.com/user-attachments/assets/95713f45-65d5-44e0-afad-979e1eb9be9b)

<details>
<summary>Allocator output</summary>

```console
$ python3 deployability/modules/allocation/main.py --action create --provider vagrant --size medium --composite-name linux-centos-10-amd64 --working-dir /home/henry/work-wazuh/requests-allocator/ --label-termination-date 1d --label-team devops --label-issue https://github.com/wazuh/wazuh/issues/24493
[2025-01-14 17:42:22] [INFO] ALLOCATOR: Creating instance at /home/henry/work-wazuh/requests-allocator
[2025-01-14 17:42:23] [DEBUG] ALLOCATOR: No config provided. Generating from payload
[2025-01-14 17:42:23] [DEBUG] ALLOCATOR: Generating new key pair
[2025-01-14 17:42:27] [DEBUG] ALLOCATOR: Vagrantfile created. Creating instance.
[2025-01-14 17:42:27] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-992 created.
[2025-01-14 17:47:36] [ERROR] ALLOCATOR: Command failed: b'Timed out while waiting for the machine to boot. This means that\nVagrant was unable to communicate with the guest machine within\nthe configured ("config.vm.boot_timeout" value) time period.\n\nIf you look above, you should be able to see the error(s) that\nVagrant had when attempting to connect to the machine. These errors\nare usually good hints as to what may be wrong.\n\nIf you\'re using a custom box, make sure that networking is properly\nworking and you\'re able to connect to the machine. It is a common\nproblem that networking isn\'t setup properly in these boxes.\nVerify that authentication configurations are also setup properly,\nas well.\n\nIf the box appears to be booting properly, you may want to increase\nthe timeout ("config.vm.boot_timeout") value.\n'
[2025-01-14 17:47:36] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-992 started.
[2025-01-14 17:47:39] [INFO] ALLOCATOR: The inventory file generated at /home/henry/work-wazuh/requests-allocator/wazuh-24493-centos-10-992/inventory.yaml
[2025-01-14 17:47:39] [INFO] ALLOCATOR: The track file generated at /home/henry/work-wazuh/requests-allocator/wazuh-24493-centos-10-992/track.yaml
[2025-01-14 17:47:42] [WARNING] ALLOCATOR: Error on attempt 1 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:48:15] [WARNING] ALLOCATOR: Error on attempt 2 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:48:48] [WARNING] ALLOCATOR: Error on attempt 3 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:49:21] [WARNING] ALLOCATOR: Error on attempt 4 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:49:54] [WARNING] ALLOCATOR: Error on attempt 5 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:50:27] [WARNING] ALLOCATOR: Error on attempt 6 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:51:00] [WARNING] ALLOCATOR: Error on attempt 7 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:51:33] [WARNING] ALLOCATOR: Error on attempt 8 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:52:06] [WARNING] ALLOCATOR: Error on attempt 9 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:52:39] [WARNING] ALLOCATOR: Error on attempt 10 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:53:13] [WARNING] ALLOCATOR: Error on attempt 11 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:53:46] [WARNING] ALLOCATOR: Error on attempt 12 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:54:19] [WARNING] ALLOCATOR: Error on attempt 13 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:54:52] [WARNING] ALLOCATOR: Error on attempt 14 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:55:25] [WARNING] ALLOCATOR: Error on attempt 15 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:55:58] [WARNING] ALLOCATOR: Error on attempt 16 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:56:31] [WARNING] ALLOCATOR: Error on attempt 17 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:57:04] [WARNING] ALLOCATOR: Error on attempt 18 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:57:37] [WARNING] ALLOCATOR: Error on attempt 19 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:58:10] [WARNING] ALLOCATOR: Error on attempt 20 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:58:43] [WARNING] ALLOCATOR: Error on attempt 21 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:59:17] [WARNING] ALLOCATOR: Error on attempt 22 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 17:59:50] [WARNING] ALLOCATOR: Error on attempt 23 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:00:23] [WARNING] ALLOCATOR: Error on attempt 24 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:00:56] [WARNING] ALLOCATOR: Error on attempt 25 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:01:29] [WARNING] ALLOCATOR: Error on attempt 26 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:02:02] [WARNING] ALLOCATOR: Error on attempt 27 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:02:35] [WARNING] ALLOCATOR: Error on attempt 28 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:03:08] [WARNING] ALLOCATOR: Error on attempt 29 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:03:41] [WARNING] ALLOCATOR: Error on attempt 30 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.93
[2025-01-14 18:04:11] [ERROR] ALLOCATOR: Connection attempts failed after 30 tries. Connection timeout.
[2025-01-14 18:04:11] [WARNING] ALLOCATOR: Rolling back instance creation.
[2025-01-14 18:04:11] [DEBUG] ALLOCATOR: Destroying instance wazuh-24493-centos-10-992
[2025-01-14 18:04:17] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-992 deleted.
```

</details>

**Comment by Enaraque** [comment]
# Update report

A `c5.metal` instance has been launched to test deploying the Stream 10 box on another machine. However, it seems that connecting to the machine correctly is not possible.

```console
ubuntu@ip-172-31-45-235:~/wazuh-automation$ python3 deployability/modules/allocation/main.py --action create --provider vagrant --size medium --composite-name linux-centos-10-amd64 --label-termination-date 1d --label-team devops --label-issue https://github.com/wazuh/wazuh/issues/24493
[2025-01-15 11:36:48] [INFO] ALLOCATOR: Creating instance at /tmp/wazuh-qa
[2025-01-15 11:36:49] [DEBUG] ALLOCATOR: No config provided. Generating from payload
[2025-01-15 11:36:49] [DEBUG] ALLOCATOR: Generating new key pair
[2025-01-15 11:36:53] [DEBUG] ALLOCATOR: Vagrantfile created. Creating instance.
[2025-01-15 11:36:53] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-1724 created.
[2025-01-15 11:42:09] [ERROR] ALLOCATOR: Command failed: b'Timed out while waiting for the machine to boot. This means that\nVagrant was unable to communicate with the guest machine within\nthe configured ("config.vm.boot_timeout" value) time period.\n\nIf you look above, you should be able to see the error(s) that\nVagrant had when attempting to connect to the machine. These errors\nare usually good hints as to what may be wrong.\n\nIf you\'re using a custom box, make sure that networking is properly\nworking and you\'re able to connect to the machine. It is a common\nproblem that networking isn\'t setup properly in these boxes.\nVerify that authentication configurations are also setup properly,\nas well.\n\nIf the box appears to be booting properly, you may want to increase\nthe timeout ("config.vm.boot_timeout") value.\n'
[2025-01-15 11:42:09] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-1724 started.
[2025-01-15 11:42:11] [INFO] ALLOCATOR: The inventory file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-1724/inventory.yaml
[2025-01-15 11:42:11] [INFO] ALLOCATOR: The track file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-1724/track.yaml
[2025-01-15 11:42:14] [WARNING] ALLOCATOR: Error on attempt 1 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:42:48] [WARNING] ALLOCATOR: Error on attempt 2 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:43:21] [WARNING] ALLOCATOR: Error on attempt 3 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:43:54] [WARNING] ALLOCATOR: Error on attempt 4 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:44:27] [WARNING] ALLOCATOR: Error on attempt 5 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:45:00] [WARNING] ALLOCATOR: Error on attempt 6 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:45:33] [WARNING] ALLOCATOR: Error on attempt 7 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:46:06] [WARNING] ALLOCATOR: Error on attempt 8 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:46:39] [WARNING] ALLOCATOR: Error on attempt 9 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:47:12] [WARNING] ALLOCATOR: Error on attempt 10 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:47:45] [WARNING] ALLOCATOR: Error on attempt 11 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:48:18] [WARNING] ALLOCATOR: Error on attempt 12 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:48:52] [WARNING] ALLOCATOR: Error on attempt 13 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:49:25] [WARNING] ALLOCATOR: Error on attempt 14 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:49:58] [WARNING] ALLOCATOR: Error on attempt 15 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:50:31] [WARNING] ALLOCATOR: Error on attempt 16 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:51:04] [WARNING] ALLOCATOR: Error on attempt 17 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:51:37] [WARNING] ALLOCATOR: Error on attempt 18 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:52:10] [WARNING] ALLOCATOR: Error on attempt 19 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:52:43] [WARNING] ALLOCATOR: Error on attempt 20 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:53:16] [WARNING] ALLOCATOR: Error on attempt 21 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:53:49] [WARNING] ALLOCATOR: Error on attempt 22 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:54:22] [WARNING] ALLOCATOR: Error on attempt 23 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:54:56] [WARNING] ALLOCATOR: Error on attempt 24 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:55:29] [WARNING] ALLOCATOR: Error on attempt 25 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:56:02] [WARNING] ALLOCATOR: Error on attempt 26 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:56:35] [WARNING] ALLOCATOR: Error on attempt 27 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:57:08] [WARNING] ALLOCATOR: Error on attempt 28 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:57:41] [WARNING] ALLOCATOR: Error on attempt 29 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:58:14] [WARNING] ALLOCATOR: Error on attempt 30 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.124
[2025-01-15 11:58:44] [ERROR] ALLOCATOR: Connection attempts failed after 30 tries. Connection timeout.
[2025-01-15 11:58:44] [WARNING] ALLOCATOR: Rolling back instance creation.
[2025-01-15 11:58:44] [DEBUG] ALLOCATOR: Destroying instance wazuh-24493-centos-10-1724
[2025-01-15 11:58:49] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-1724 deleted.
```

**Comment by Enaraque** [comment]
I am marking the issue as blocked until an instance suitable for creating an AMI in AWS is available. The Vagrant box from the official site will be re-imported, and the process will be tested again.

**Comment by fcaffieri** [comment]
### Update report

Vagrant Box has a recent update, but the same problem has occurred.
It throws this error:

```
$ vagrant up stack-centos10
Bringing machine 'stack-centos10' up with 'virtualbox' provider...
==> stack-centos10: Importing base box 'centos/stream10'...
==> stack-centos10: Matching MAC address for NAT networking...
==> stack-centos10: Checking if box 'centos/stream10' version '20250123.0' is up to date...
==> stack-centos10: Setting the name of the VM: vagrant_stack_stack-centos10_1738697062077_74194
==> stack-centos10: Fixed port collision for 22 => 2222. Now on port 2200.
==> stack-centos10: Clearing any previously set network interfaces...
==> stack-centos10: Preparing network interfaces based on configuration...
    stack-centos10: Adapter 1: nat
    stack-centos10: Adapter 2: hostonly
==> stack-centos10: Forwarding ports...
    stack-centos10: 1514 (guest) => 1529 (host) (adapter 1)
    stack-centos10: 22 (guest) => 2200 (host) (adapter 1)
==> stack-centos10: Running 'pre-boot' VM customizations...
==> stack-centos10: Booting VM...
==> stack-centos10: Waiting for machine to boot. This may take a few minutes...
    stack-centos10: SSH address: 127.0.0.1:2200
    stack-centos10: SSH username: vagrant
    stack-centos10: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.

```

I tried increasing the timeout, but the VM throws the error:

![Image](https://github.com/user-attachments/assets/ab8ee806-52c3-47c6-9e2a-b8c19943f34c)

**Comment by fcaffieri** [comment]
### New tests:


Tried with the official CentOS 10 box and with the bento box, getting the same kernel panic error on both

CentOS 10 Vagrant:

```
$ vagrant up stack-centos10
Bringing machine 'stack-centos10' up with 'virtualbox' provider...
==> stack-centos10: Box 'bento/centos-stream-10' could not be found. Attempting to find and install...
    stack-centos10: Box Provider: virtualbox
    stack-centos10: Box Version: >= 0
==> stack-centos10: Loading metadata for box 'bento/centos-stream-10'
    stack-centos10: URL: https://vagrantcloud.com/api/v2/vagrant/bento/centos-stream-10
==> stack-centos10: Adding box 'bento/centos-stream-10' (v202502.21.0) for provider: virtualbox (amd64)
    stack-centos10: Downloading: https://vagrantcloud.com/bento/boxes/centos-stream-10/versions/202502.21.0/providers/virtualbox/amd64/vagrant.box
==> stack-centos10: Successfully added box 'bento/centos-stream-10' (v202502.21.0) for 'virtualbox (amd64)'!
==> stack-centos10: Importing base box 'bento/centos-stream-10'...
==> stack-centos10: Matching MAC address for NAT networking...
==> stack-centos10: Checking if box 'bento/centos-stream-10' version '202502.21.0' is up to date...
==> stack-centos10: Setting the name of the VM: vagrant_stack_stack-centos10_1740516049624_39302
==> stack-centos10: Clearing any previously set network interfaces...
==> stack-centos10: Preparing network interfaces based on configuration...
    stack-centos10: Adapter 1: nat
    stack-centos10: Adapter 2: hostonly
==> stack-centos10: Forwarding ports...
    stack-centos10: 1514 (guest) => 1529 (host) (adapter 1)
    stack-centos10: 22 (guest) => 2222 (host) (adapter 1)
==> stack-centos10: Running 'pre-boot' VM customizations...
==> stack-centos10: Booting VM...
==> stack-centos10: Waiting for machine to boot. This may take a few minutes...
    stack-centos10: SSH address: 127.0.0.1:2222
    stack-centos10: SSH username: vagrant
    stack-centos10: SSH auth method: private key
    stack-centos10: Warning: Connection reset. Retrying...

```

When testing with AWS, it seems that the problem is the same, it is not possible to connect via ssh

CentOS 10 AWS:

```
$ python3 deployability/modules/allocation/main.py --provider aws --size large --composite-name linux-centos-10-amd64 --label-termination-date 1d --label-team devops --label-issue https://github.com/wazuh/wazuh/issues/24493 --aws-account wazuh-dev --aws-profile wazuh-dev
[2025-02-25 17:42:30] [INFO] ALLOCATOR: Creating instance at /tmp/wazuh-qa
[2025-02-25 17:42:31] [DEBUG] ALLOCATOR: No config provided. Generating from payload
[2025-02-25 17:42:31] [DEBUG] ALLOCATOR: Generating new key pair
[2025-02-25 17:42:31] [DEBUG] ALLOCATOR: Creating base directory: /tmp/wazuh-qa/AWS-D29CAEF2-CA25-47FD-A003-36BD7E4BF3F8
[2025-02-25 17:42:52] [DEBUG] ALLOCATOR: Renaming temp /tmp/wazuh-qa/AWS-D29CAEF2-CA25-47FD-A003-36BD7E4BF3F8 directory to /tmp/wazuh-qa/wazuh-24493-centos-10-8283
[2025-02-25 17:42:52] [INFO] ALLOCATOR [WAZUH-24493-CENTOS-10]: Instance i-02fa2e400b169fe8d created.
[2025-02-25 17:42:54] [INFO] ALLOCATOR [WAZUH-24493-CENTOS-10]: Instance i-02fa2e400b169fe8d started.
[2025-02-25 17:42:54] [INFO] ALLOCATOR [WAZUH-24493-CENTOS-10]: The inventory file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-8283/inventory.yaml
[2025-02-25 17:42:54] [INFO] ALLOCATOR [WAZUH-24493-CENTOS-10]: The track file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-8283/track.yaml
[2025-02-25 17:45:04] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 1 of 30: [Errno 110] Connection timed out
[2025-02-25 17:45:34] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 2 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:46:04] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 3 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:46:34] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 4 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:47:05] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 5 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:47:35] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 6 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:48:05] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 7 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
[2025-02-25 17:48:35] [WARNING] ALLOCATOR [WAZUH-24493-CENTOS-10]: Error on attempt 8 of 30: [Errno None] Unable to connect to port 2200 on 18.207.239.104
```

It continues in blocks, until a stable version is obtained.

**Comment by c-bordon** [comment]
The problem in Vagrant persists:

```console
cbordon@cbordon-MS-7C88:~/Documents/wazuh/repositorios/wazuh-automation$ python3 deployability/modules/allocation/main.py --provider vagrant --size large --label-issue https://github.com/wazuh/wazuh/issues/24493 --composite-name linux-centos-10-amd64
[2025-04-03 14:17:45] [INFO] ALLOCATOR: Creating instance at /tmp/wazuh-qa
[2025-04-03 14:17:46] [DEBUG] ALLOCATOR: No config provided. Generating from payload
[2025-04-03 14:17:46] [DEBUG] ALLOCATOR: Generating new key pair
[2025-04-03 14:17:49] [DEBUG] ALLOCATOR: Vagrantfile created. Creating instance.
[2025-04-03 14:17:49] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-5191 created.
[2025-04-03 14:24:59] [ERROR] ALLOCATOR: Command failed: b'==> vagrant: A new version of Vagrant is available: 2.4.3 (installed version: 2.3.4)!\n==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html\n\nTimed out while waiting for the machine to boot. This means that\nVagrant was unable to communicate with the guest machine within\nthe configured ("config.vm.boot_timeout" value) time period.\n\nIf you look above, you should be able to see the error(s) that\nVagrant had when attempting to connect to the machine. These errors\nare usually good hints as to what may be wrong.\n\nIf you\'re using a custom box, make sure that networking is properly\nworking and you\'re able to connect to the machine. It is a common\nproblem that networking isn\'t setup properly in these boxes.\nVerify that authentication configurations are also setup properly,\nas well.\n\nIf the box appears to be booting properly, you may want to increase\nthe timeout ("config.vm.boot_timeout") value.\n'
[2025-04-03 14:24:59] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-5191 started.
[2025-04-03 14:25:02] [INFO] ALLOCATOR: The inventory file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-5191/inventory.yaml
[2025-04-03 14:25:02] [INFO] ALLOCATOR: The track file generated at /tmp/wazuh-qa/wazuh-24493-centos-10-5191/track.yaml
[2025-04-03 14:25:05] [WARNING] ALLOCATOR: Error on attempt 1 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:25:38] [WARNING] ALLOCATOR: Error on attempt 2 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:26:11] [WARNING] ALLOCATOR: Error on attempt 3 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:26:44] [WARNING] ALLOCATOR: Error on attempt 4 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:27:17] [WARNING] ALLOCATOR: Error on attempt 5 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:27:50] [WARNING] ALLOCATOR: Error on attempt 6 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:28:23] [WARNING] ALLOCATOR: Error on attempt 7 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:28:56] [WARNING] ALLOCATOR: Error on attempt 8 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:29:29] [WARNING] ALLOCATOR: Error on attempt 9 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:30:03] [WARNING] ALLOCATOR: Error on attempt 10 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:30:36] [WARNING] ALLOCATOR: Error on attempt 11 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:31:09] [WARNING] ALLOCATOR: Error on attempt 12 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:31:42] [WARNING] ALLOCATOR: Error on attempt 13 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:32:15] [WARNING] ALLOCATOR: Error on attempt 14 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:32:48] [WARNING] ALLOCATOR: Error on attempt 15 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:33:21] [WARNING] ALLOCATOR: Error on attempt 16 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:33:54] [WARNING] ALLOCATOR: Error on attempt 17 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:34:27] [WARNING] ALLOCATOR: Error on attempt 18 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:35:00] [WARNING] ALLOCATOR: Error on attempt 19 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:35:33] [WARNING] ALLOCATOR: Error on attempt 20 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:36:06] [WARNING] ALLOCATOR: Error on attempt 21 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:36:40] [WARNING] ALLOCATOR: Error on attempt 22 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:37:13] [WARNING] ALLOCATOR: Error on attempt 23 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:37:46] [WARNING] ALLOCATOR: Error on attempt 24 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:38:19] [WARNING] ALLOCATOR: Error on attempt 25 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:38:52] [WARNING] ALLOCATOR: Error on attempt 26 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:39:25] [WARNING] ALLOCATOR: Error on attempt 27 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:39:58] [WARNING] ALLOCATOR: Error on attempt 28 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:40:31] [WARNING] ALLOCATOR: Error on attempt 29 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:41:04] [WARNING] ALLOCATOR: Error on attempt 30 of 30: [Errno None] Unable to connect to port 22 on 192.168.57.91
[2025-04-03 14:41:34] [ERROR] ALLOCATOR: Connection attempts failed after 30 tries. Connection timeout.
[2025-04-03 14:41:34] [WARNING] ALLOCATOR: Rolling back instance creation.
[2025-04-03 14:41:34] [DEBUG] ALLOCATOR: Destroying instance wazuh-24493-centos-10-5191
[2025-04-03 14:41:39] [INFO] ALLOCATOR: Instance wazuh-24493-centos-10-5191 deleted.
```

![Image](https://github.com/user-attachments/assets/79351627-5a74-4440-89ab-0a18155da35a)

https://portal.cloud.hashicorp.com/vagrant/discover/centos/stream10

vagrantbox: `centos/stream10`
version: `20250331.0
